src/frontend/js/data/index.tsx

```diff
@@ -122,6 +122,7 @@ export type UseReferralLitesParams = {
   unit?: string[];
   user?: string[];
```

Comment:
You have actually removed this parameter from the API so you should also remove it from the type definition.

Reply:
This makes sense but then we cannot remove it from the API.